schema/gene.js

```diff
@@ -57,7 +57,14 @@ const GeneType = new GraphQLObjectType({
       },
     },
     artworks_connection: {
-      type: artworkConnection,
+      type: connectionDefinitions({
+        name: "GeneArtworks",
+        nodeType: Artwork.type,
+        connectionFields: {
```

Comment:
So this just adds these two fields, `aggregations` and `counts`, inside of each `edge` field in the connection?

Reply:
Almost, these are in the root of the connection, next to `edges`.

Reply:
Ah, yes- that's what I meant!